src/org/pentaho/platform/dataaccess/datasource/wizard/service/messages/messages.properties

```diff
@@ -118,6 +118,8 @@ InMemoryDatasourceServiceImpl.ERROR_0020_UNABLE_TO_GET_DATASOURCE=Unable to cons
 
 ConnectionServiceHelper.ERROR_0001_UNABLE_TO_GET_CONNECTION_PASSWORD=Unable to get the connection password: {0}
 
+AnalysisService.ERROR_001_ANALYSIS_DATASOURCE_ERROR=Error creating Analysis datasource. Check the provided XML file.
```

Comment:
This message should be more explicit. The only way it gets triggered is if the extension was .xmi. The message should reflect that.